openimis.json

```diff
@@ -134,10 +134,6 @@
       "name": "DeduplicationModule",
       "npm": "@openimis/fe-deduplication@git+https://github.com/openimis/openimis-fe-deduplication_js.git#develop"
     },
-    {
-      "name": "GrievanceSocialProtectionModule",
-      "npm": "@openimis/fe-grievance_social_protection@git+https://github.com/openimis/openimis-fe-grievance_social_protection_js.git#develop"
-    },
```

Comment:
@hirensoni913 why was GrievanceSocialProtectionModule removed? It exists in the backend.